pkg/repo/fs.go

```diff
@@ -213,6 +217,9 @@ func (fsr *FsRepo) Init() error {
 	if err != nil {
 		return err
 	}
+
+	// TODO this should be a part of the config.
+	telemetry.SetupFromEnvs()
```

Comment:
Does this need to be in Open _and_ Init?

Reply:
I think our conclusion was maybe? And we're not confident enough to consolidate this.